build.md

```diff
@@ -112,14 +112,27 @@ build:
 
 `dockerfile` allows to set an alternate Dockerfile. A relative path MUST be resolved from the build context.
 Compose implementations MUST warn user about absolute path used to define Dockerfile as those prevent Compose file
-from being portable.
+from being portable. When set, `dockerfile_inline` attribute is not allowed.
```

Comment:
Maybe we should be clear on the behaviour expected when this case will happen? Should the Compose runtime refuse the Compose file as invalid or warn the user that the `dockerfile` will be used instead of the `inline` definition?

Reply:
I read it before as an implementation should refuse a Compose file that has both (since it's not allowed), but we could make that clearer 👍